pkg/kubelet/pod_workers.go

```diff
@@ -61,6 +63,12 @@ type podWorkers struct {
 
 	// The EventRecorder to use
 	recorder record.EventRecorder
+
+	// backOffInterval is the duration to back off when there is a sync error.
+	backOffInterval time.Duration
```

Comment:
(nit) Per https://github.com/kubernetes/kubernetes/issues/12866#issuecomment-132388570 "period" is prefered to "interval"